Source/Classes/PINRemoteImageManager.m

```diff
@@ -97,6 +97,32 @@ BOOL PINRemoteImageManagerSubclassOverridesSelector(Class subclass, SEL selector
 NSString * const PINRemoteImageCacheKeyResumePrefix = @"R-";
 typedef void (^PINRemoteImageManagerDataCompletion)(NSData *data, NSURLResponse *response, NSError *error);
 
+@interface PINRemoteImageWeakContainer : NSObject
+
+@property (nonatomic, readonly, weak) id target;
+
++ (PINRemoteImageWeakContainer *)weakContainerWithTarget:(id)target;
```

Comment:
Because this is only used internally, we could save the extra code + symbols by removing this class method and just use `-initWithTarget:` in the one place we call it.